docs/_config.yml

```diff
@@ -18,10 +18,15 @@ docs-jms-connections: http://docs.solace.com/Solace-JMS-API/Establishing-Connect
 docs-jms-managing-sessions: http://docs.solace.com/Solace-JMS-API/Managing-Sessions.htm
 docs-jms-obtaining-connection-factories: http://docs.solace.com/Solace-JMS-API/Obtaining-Connection-Fac.htm
 docs-jms-working-with-destinations: http://docs.solace.com/Solace-JMS-API/Working-with-Destination.htm
+docs-jms-establishing-connections: http://docs.solace.com/Solace-JMS-API/Obtaining-Connection-Fac.htm#establishing_connections_577400906_317143
```

Comment:
This link will break. To link here, we need to work with the Solace docs team to remove: _577400906_317143